lib/hexo.js

```diff
@@ -86,7 +86,7 @@ entry.version = require('../package.json').version;
 
 function loadModule(path, args) {
   return Promise.try(function() {
-    var modulePath = pathFn.join(path, 'node_modules', 'hexo');
+    var modulePath = resolve.sync('hexo', { basedir: path });
```

Comment:
It is sufficient to use builtin [`require.resolve`](https://nodejs.org/api/modules.html#modules_all_together) since `resolve` implements `require.resolve` for browser-side support and `hexo` does not, at least officially, support browser environment.

Reply:
Fair enough! At the time I discovered the `resolve` package, node didn't export the right stuff. I've patched it!